app/src/main/kotlin/com/simplemobiletools/smsmessenger/receivers/MmsSentReceiver.kt

```diff
@@ -19,6 +22,7 @@ class MmsSentReceiver : SendStatusReceiver() {
         val messageBox = if (receiverResultCode == Activity.RESULT_OK) {
             Telephony.Mms.MESSAGE_BOX_SENT
         } else {
+            context.toast(msg = context.getString(R.string.unknown_error_occurred_sending_message, receiverResultCode), length = Toast.LENGTH_LONG)
```

Comment:
why is receiverResultCode needed here? Theres too many things going on on this line

Reply:
> why is receiverResultCode needed here? 

it'll be shown with "Couldn't send message, error code: %s". The error code can help us debug the exact error from SmsManager

> Theres too many things going on on this line

fixed

Reply:
But we can add more error-specific strings based on the error code as we know what they mean